application/main.py

```diff
@@ -794,6 +794,30 @@ def get_model(fn):
         return response
     else:
         return send_file(path)
+    
+@application.route('/api/bsdd/statistics/<id>', methods=['GET'])
```

Comment:
I would prefer for coherence to have `/api/bsdd_statistics/<id>`  instead of `/api/bsdd/statistics/<id>`

Reply:
Done https://github.com/Ghesselink/ifc-pipeline-validation/commit/b6a316e38e91bcb2c6602bce57312b58b8160239